overrides/groovy/postInit/Post-Initial/Main/General/Misc/materialChanges.groovy

```diff
@@ -10,16 +10,16 @@ material('rhodium_plated_palladium')
 	.changeChemicalFormula()
 	.change()
 
+/* Black Steel */
+
 // Can't use change composition to remove, as that is only performed at the end of running scripts, and if not removed, conflicts will occur.
 // TODO Make this use change composition when that accepts non-material Item Stacks
```

Comment:
Everything looks good, Only comment is if the TODO is quick, add it to the pr, other then that, should be mergable

Reply:
Yeah that TODO is... quite far from complete, and requires some labs helpers which will probably come later. Also doubting if replace composition is actually useful, given its limited use case, but that can be decided later.

Reply:
Ok then you can merge it